lib/liquid/standardfilters.rb

```diff
@@ -160,9 +160,9 @@ def where(input, property, target_value = nil)
       if ary.empty?
         []
       elsif ary.first.respond_to?(:[]) && target_value.nil?
-        ary.where_present(property)
+        ary.select { |item| get_property(item, property) }
```

Comment:
Could we move this logic into `where` / `where_present` instead of here?

Reply:
100%, these methods are near the bottom of the file on `InputIterator`, not part of stdlib.

Reply:
As-is, this PR deletes those methods, which I think is fine now that `get_property` contains the logic.